src/Indicator.vala

```diff
@@ -100,12 +106,61 @@ public class Session.Indicator : Wingpanel.Indicator {
             lock_screen.get_child ().destroy ();
             lock_screen.add (lock_screen_grid);
 
-            shutdown = new Gtk.ModelButton ();
-            shutdown.hexpand = true;
-            shutdown.text = _("Shut Down…");
-
-            suspend = new Gtk.ModelButton ();
-            suspend.text = _("Suspend");
+            shutdown = new Gtk.Button ();
+            shutdown.halign = Gtk.Align.CENTER;
+            var shutdown_label = new Gtk.Label (_("<small>Shut Down…</small>"));
```

Comment:
You're including the `<small>` tag in the translatable string, which we should probably avoid (more room for human error changing the layout with translations). Instead, we should probably use a style class or at least separate the markup from the copy with a `printf ()`.